src/Google/Service/AndroidEnterprise/Resource/Enterprises.php

```diff
@@ -126,19 +126,6 @@ public function get($enterpriseId, $optParams = array())
     $params = array_merge($params, $optParams);
     return $this->call('get', array($params), "Google_Service_AndroidEnterprise_Enterprise");
   }
-  /**
-   * Deprecated and unused. (enterprises.getAndroidDevicePolicyConfig)
-   *
-   * @param string $enterpriseId The ID of the enterprise.
-   * @param array $optParams Optional parameters.
-   * @return Google_Service_AndroidEnterprise_AndroidDevicePolicyConfig
-   */
-  public function getAndroidDevicePolicyConfig($enterpriseId, $optParams = array())
```

Comment:
concerning that this would cause a BC break, as usually this doesn't happen with these updates

Reply:
This is definitely missing from the latest version of the discovery doc, what recourse would we have? Hopefully it is okay since it does note that it is deprecated.